src/Command/CloudAccount/Backup/Delete.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @package Nexcess-CLI
+ * @license https://opensource.org/licenses/MIT
+ * @copyright 2018 Nexcess.net, LLC
+ */
+
+declare(strict_types = 1);
+
+namespace Nexcess\Sdk\Cli\Command\CloudAccount\Backup;
+
+use Closure;
+
+use Nexcess\Sdk\ {
+  Resource\CloudAccount\Backup,
+  Resource\CloudAccount\Endpoint,
+  Resource\Promise,
+  Util\Config,
+  Util\Util
+};
+
+use Nexcess\Sdk\Cli\ {
+  Command\CloudAccount\CloudAccountException,
+  Command\Create as CreateCommand,
+  Console
+};
+
+use Symfony\Component\Console\ {
+  Input\InputArgument as Arg,
+  Input\InputInterface as Input,
+  Input\InputOption as Opt,
+  Output\OutputInterface as Output
+};
+
+/**
+ * Creates a new Cloud Account.
+ */
+class Delete extends CreateCommand {
```

Comment:
shouldn't extend from CreateCommand.  there is presently no abstract DeleteCommand, but there probably should be since most delete actions follow the same form.

however, since backups are the exception to every rule (Backup\Delete would have to override a generic Delete command anyway), we could extend from `Command` for now and re-assess later…?